nixos/doc/manual/configuration/xfce.chapter.md

```diff
@@ -54,4 +54,4 @@ run this command to do the same thing.
 $ xfconf-query -c xfce4-session -p /compat/LaunchGNOME -s true
 ```
 
-A log-out and re-log will be needed for this to take effect.
+It is necessary to logout and login again for this to take effect.
```

Comment:
Nitpicking the nitpick: ["login" is a noun, "log in" is a verb](http://www.notaverb.com/login).
```suggestion
It is necessary to log out and log in again for this to take effect.
```